code/modules/admin/verbs/adminpm.dm

```diff
@@ -186,16 +186,16 @@
 
 				//AdminPM popup for ApocStation and anybody else who wants to use it. Set it with POPUP_ADMIN_PM in config.txt ~Carn
 				if(config.popup_admin_pm)
-					spawn()	//so we don't hold the caller proc up
-						var/sender = src
-						var/sendername = key
-						var/reply = input(recipient, msg,"Admin PM from-[sendername]", "") as text|null		//show message and await a reply
-						if(recipient && reply)
-							if(sender)
-								recipient.cmd_admin_pm(sender,reply)										//sender is still about, let's reply to them
-							else
-								adminhelp(reply)													//sender has left, adminhelp instead
-						return
+					set waitfor = FALSE	//so we don't hold the caller proc up

```

Comment:
this being allowed is a byond bug.

It is not allowed to `set` things after the first line in a proc, 

And I tested, this will apply to the entire proc, not this if section